src/storage/log/mod.rs

```diff
@@ -712,6 +712,12 @@ fn merge_slices(dest: &mut [u8], src: &[u8]) -> usize {
     min_len
 }
 
+fn copy_into_dest_from_src(dest: &mut [u8], dest_len: usize, src: &[u8], src_len: usize) -> usize {
```

Comment:
Nit but either naming this so that it is clear that "it tries to copy dest.len bytes. but no more than src_len, from src"(hope it makes sense 😄), or add a comment to this function to explain that would be easier to read!

Reply:
Ack! Will do so

Reply:
Added comment, lemme know if this is ok